src/tests/test_utils/functional_test_utils/CMakeLists.txt

```diff
@@ -29,6 +29,10 @@ addIeTarget(
             $<TARGET_PROPERTY:openvino::runtime::dev,INTERFACE_INCLUDE_DIRECTORIES>
 )
 
+install(FILES layer_tests_summary/run_parallel.py DESTINATION tests/functional_test_utils COMPONENT tests EXCLUDE_FROM_ALL)
```

Comment:
```suggestion
install(PROGRAMS layer_tests_summary/run_parallel.py DESTINATION tests/functional_test_utils COMPONENT tests EXCLUDE_FROM_ALL)
```
https://cmake.org/cmake/help/latest/command/install.html#programs

Reply:
we don't need an execution permission for python scripts, changed to be aligned